src/components/ProjetoDialog.tsx

```
@@ -44,22 +94,19 @@ export function ProjetoDialog({
   const handleCandidatura = async () => {
     try {
-      // Verifica se o projeto tem vagas disponíveis
-     
+      // Chamar o endpoint de notificação quando o usuário se candidatar
 
-      // Verifica se o aluno já está inscrito
-      
-      // Envia a candidatura
-     toast.success("Candidatura enviada com sucesso!");
+
+      toast.success("Candidatura enviada com sucesso!");
```

ESLint

[error] Replace `⏎······toast.success("Candidatura·enviada·com·sucesso!");` with `······toast.success('Candidatura·enviada·com·sucesso!')`

(prettier/prettier)